src/structures/Guild.js

```diff
@@ -1387,16 +1381,7 @@ class Guild extends Base {
    * @deprecated
    */
   setEmbed(embed, reason) {
```

Comment:
Ditto

Reply:
https://github.com/discordjs/discord.js/blob/405b487dc3475ea077dcd3f6734a5a01321021dd/src/structures/Guild.js#L1548-L1550

Reply:
I am not blind, I promise 😅